docs/UserGuide.md

```diff
@@ -321,47 +330,72 @@ find David
 </figure>
 <br>
 
+[↑ Back to Table of Contents](#table-of-contents)
 
-### Viewing the full details of a client: ***view***
-
-Catching up on the details of a client before contacting them is important. Most of the time, remembering the details of previous interactions with a client is difficult as the data is scattered everywhere.
+### Viewing the dashboard: ***dashboard***
 
-Well, not anymore! With Connectify, you can view the full details of a client and past interactions in a single place using the ***view*** command.
+Use the ***dashboard*** command to view a summarised information of all your clients and their interactions with you in a single place.
 
 **Format**
-```text
-view INDEX
-```
+````text
+dashboard
+````
 
-<div markdown="block" class="alert alert-info">
+You should see the following dashboard view in the application window.
 
-**:information_source: Notes about the view command**<br>
+<figure>
+   <img src="images/Dashboard.png" alt="Dashboard view">
+   <figcaption align="center">
+       <em>The dashboard view. Data shown may vary.</em>
+   </figcaption>
+</figure>
+<br>
 
-`INDEX` refers to the index of the client profile in the displayed list of clients.
+On the left, we have Connectify's statistics which shows:
 
-</div>
+1. Uncontacted / Contacted / Closed clients
+2. Average interactions amongst all clients
+2. Breakdown of the different interaction outcomes
+3. Number of HOT / WARM / COLD leads
 
-**Example**
+On the right, we have the follow-ups section which shows upcoming follow-ups. Follow-up dates are determined automatically by the latest interaction date and the lead of the client.
+
+<details>
+<summary>Detailed explanation of follow-up calculation</summary>
+<br>
+The follow up date is calculated by the day of last interaction plus a follow-up period, determined by lead type as shown:
+
+| LeadType | Follow-Up Time |
+|----------|----------------|
+| HOT      | 1 week         |
+| WARM     | 4 weeks        |
+| COLD     | 8 weeks        |
+| UNKNOWN  | 4 weeks        |
+
+
+Say your last interaction with John is on 1st Jan 2021 and he is a HOT lead. The follow-up date will be 1 week after 1st Jan 2021, which is 8th Jan 2021. 
+</details>
+
+[↑ Back to Table of Contents](#table-of-contents)
+
+### Viewing the list of clients: ***list***
```

Comment:
Would it be better to have list of clients above as before? We can only see the clients after going from dashboard to list

Reply:
Okay true...